backend/Controllers/SightingsController.cs

```diff
@@ -26,9 +26,32 @@ public IActionResult GetBySightingId([FromRoute] int sightingId)
     {
         try
         {
-            var sightingResult = _sightingService.GetSightingById(sightingId);
-            Sighting sighting = sightingResult.Result;
-            return Ok(sighting);
+            int userId = int.Parse(User.FindFirst(ClaimTypes.NameIdentifier).Value);
+            var singlesighting = _sightingService.GetSightingById(sightingId);
+            SingleSightingResponse sighting = new SingleSightingResponse()
+            {
+                Id = singlesighting.Result.Id,
+                UserId = singlesighting.Result.UserId,
+                SpeciesId = singlesighting.Result.SpeciesId,
+                Latitude = singlesighting.Result.Latitude,
+                Longitude = singlesighting.Result.Longitude,
+                PhotoUrl = singlesighting.Result.PhotoUrl,
+                Description = singlesighting.Result.Description,
+                DateTime = singlesighting.Result.DateTime,
+                IsApproved = singlesighting.Result.IsApproved,
+            };
```

Comment:
This logic should be happening in the sightings service rather than the controller.

As a general rule the controller should receive requests and send out responses  but it shouldn't be involved in making the responses.

Reply:
done